src/test/scala/com/bazaarvoice/sswf/ListOpenExecutionSteps.java

```diff
@@ -0,0 +1,25 @@
+package com.bazaarvoice.sswf;
+
+public enum ListOpenExecutionSteps implements WorkflowStep {
```

Comment:
Maybe we can call this `ListOpenExecutionTestSteps` so it's clear that it's for testing